website/docs/installation/docker-compose.md

```diff
@@ -49,55 +49,57 @@ AUTHENTIK_EMAIL__TIMEOUT=10
 AUTHENTIK_EMAIL__FROM=authentik@localhost
 ```
 
-## Running on Port 80/443
+## Configure for port 80/443
 
-By default, authentik listens on port 9000 for HTTP and 9443 for HTTPS. To change this, you can set the following variables in `.env`:
+By default, authentik listens on port 9000 for HTTP and 9443 for HTTPS. To change the default and instead use ports 80 and 443, you can set the following variables in `.env`:
 
 ```shell
 AUTHENTIK_PORT_HTTP=80
 AUTHENTIK_PORT_HTTPS=443
 ```
 
-Afterwards, make sure to run `docker-compose up -d`.
+Be sure to run `docker-compose up -d` to rebuild with the new port numbers.
 
 ## Startup
 
-Afterwards, run these commands to finish
+Afterwards, run these commands to finish:
 
 ```shell
 docker-compose pull
 docker-compose up -d
 ```
 
-The compose file statically references the latest version available at the time of downloading the compose file, which can be overridden with the `AUTHENTIK_TAG` environment variable.
+The `docker-compose.yml` file statically references the latest version available at the time of downloading the compose file, which can be overridden with the `AUTHENTIK_TAG` environment variable.
 
-authentik will then be reachable on port 9000 (HTTP) and port 9443 (HTTPS).
+authentik is then reachable (by default) on port 9000 (HTTP) and port 9443 (HTTPS).
 
-To start the initial setup, navigate to `https://<your server>/if/flow/initial-setup/`. There you will be prompted to set a password for the akadmin user.
+To start the initial setup, navigate to `https://<your server>/if/flow/initial-setup/`. If you are using your local host, this will be `http://localhost:9000/if/flow/initial-setup/`.
```

Comment:
```suggestion
To start the initial setup, navigate to `https://<your server's IP or hostname>:9000/if/flow/initial-setup/`.
```

I think very few people will use these instructions to run authentik on the machine they are accessing it from, and those people are most likely doing so for a reason hence I think it makes sense to assume they know how to access it

Reply:
agree, good point.